server/src/main/xsd/gateway-config-201409.xsd

```diff
@@ -1253,6 +1253,16 @@
                     </documentation>
                 </annotation>
             </element>
+            <element name="http.keepalive.max.connections" maxOccurs="1" minOccurs="0" type="nonNegativeInteger" default="5">
```

Comment:
@robinzimmermann can you review the name. It is something equivalent to nginx 's keepalive connections config. see here http://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive